components/scripts/lib/build-scan-online.sh

```diff
@@ -53,28 +53,27 @@ fetch_single_build_scan() {
   parse_single_build_scan "${build_scan_data}"
 }
 
+# The value of logging_level should be either 'brief_logging' or
+# 'verbose_logging'
 fetch_build_scans_and_build_time_metrics() {
-  local build_cache_metrics_only="$1"
+  local logging_level="$1"
   shift
   local build_scan_urls=("$@")
 
-  local brief_logging
-  if [[ "${build_cache_metrics_only}" == 'build_cache_metrics_only' ]]; then
-    brief_logging="brief_logging"
-  else
+  if [[ "${logging_level}" != 'brief_logging' ]]; then
```

Comment:
🧐 You could reverse this to be a positive test for "verbose_logging" instead. Positive tests tend to be easier to understand.

Reply:
That is a good point. Done in 2428dacd36f1f86a4c9f8ca66a2f36c1efa05037.